components/brave_wallet_ui/common/slices/wallet.slice.ts

```diff
@@ -260,6 +262,8 @@ export const WalletAsyncActions = {
   // alias for keyringService.addFilecoinAccount
   addFilecoinAccount:
     createAction<AddFilecoinAccountPayloadType>('addFilecoinAccount'),
+  addBitcoinAccount:
```

Comment:
please move this to a mutation within `api.slice.ts` so we can update the account cache when this action is dispatched